ext/opentelemetry-ext-azure-monitor/examples/azure_requests.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import requests
+from opentelemetry import trace
+from opentelemetry.ext import http_requests
+from opentelemetry.ext.azure_monitor import AzureMonitorSpanExporter
```

Comment:
We are deprecating opentelemetry.ext.azure_monitor in favor of opentelemetry-azure-monitor-exporter

https://github.com/open-telemetry/opentelemetry-python/pull/272